Makefile

```
@@ -75,15 +80,24 @@ syntax-docs: docgen
 
 integration:
-	cd integration_tests; bash run.sh
-
-functional:
-	cd cmd/functional-test; bash run.sh
+	$(GOTEST) -tags=integration -timeout 1h ./internal/tests/integration
+
+integration-debug:
+	$(GOTEST) -tags=integration ./internal/tests/integration -v $(GO_TEST_ARGS) -args $(INTEGRATION_ARGS)
+
+functional: build
```

checkmake (0.2.2)

[warning] Target body for "functional" exceeds allowed length of 5 (7).

(maxbodylength)